lib/gringotts/gateways/authorize_net.ex

```diff
@@ -565,4 +565,13 @@ defmodule Gringotts.Gateways.AuthorizeNet do
   defp join_string(list, symbol) do
     Enum.join(list, symbol)
   end
+  
+  defp base_url(opts) do
+    cond do
+      opts[:config][:mode] == "prod" -> @production_url
```

Comment:
Instead of using the strings `"prod"` and `"test"`, let's use atoms `:prod` and `:test`. Feels more elixir-like 😃

Reply:
good point!

Reply:
Yeah I think it's a mistake on my part, earlier I was thinking of passing this in the opts. But, now that it's being set in the config it should be an atom.